pkg/backup/backup_compaction_test.go

```diff
@@ -313,6 +313,30 @@ crdb_internal.json_to_pb(
 		db.Exec(t, "CANCEL JOB $1", jobID)
 		jobutils.WaitForJobToCancel(t, db, jobID)
 	})
+
+	t.Run("builtin using backup statement", func(t *testing.T) {
+		db.Exec(t, "CREATE TABLE foo (a INT, b INT)")
+		defer func() {
+			db.Exec(t, "DROP TABLE foo")
+		}()
+		db.Exec(t, "INSERT INTO foo VALUES (1, 1)")
+		start := getTime(t)
+		opts := "encryption_passphrase = 'correct-horse-battery-staple'"
+		db.Exec(t, fmt.Sprintf(fullBackupAostCmd+" WITH %s", 9, start, opts))
+		db.Exec(t, "INSERT INTO foo VALUES (2, 2)")
+		db.Exec(t, fmt.Sprintf(incBackupCmd+" WITH %s", 9, opts))
+		db.Exec(t, "INSERT INTO foo VALUES (3, 3)")
+		end := getTime(t)
+		db.Exec(t, fmt.Sprintf(incBackupAostCmd+" WITH %s", 9, end, opts))
+		var jobID jobspb.JobID
+		db.QueryRow(t, fmt.Sprintf(
+			`SELECT crdb_internal.backup_compaction(
+        'BACKUP INTO LATEST IN ''nodelocal://1/backup/9'' WITH encryption_passphrase = ''correct-horse-battery-staple''', 
+        %d::DECIMAL, %d::DECIMAL
+      )`, start, end,
+		)).Scan(&jobID)
+		waitForSuccessfulJob(t, tc, jobID)
```

Comment:
validate the the data is correct as well? to ensure all the timestamps were passed correctly?